auth/user.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
We do not need auth data under the git control, move auth folder to the .gitignore file